catalog/plugin/catalog-plugin-metacardbackup-filestorage/src/main/java/org/codice/ddf/catalog/plugin/metacard/backup/storage/filestorage/MetacardFileStorageRoute.java

```diff
@@ -0,0 +1,110 @@
+/**
```

Comment:
I am fine with this, but we may get some pushback to do this all in blueprint.  I think we it could be done, but it would be a readability nightmare